src/main/java/core/basesyntax/GeometricFigure.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax;
+
+public enum GeometricFigure {

```

Comment:
lets not confuse others with same names)
```suggestion
public enum FigureName {
```

Reply:
Ok